### PR TITLE
Add content to Connect screen

### PR DESCRIPTION
**Pull request by devinjameson**
Why: we don't have any content on the Connect screen right now.

This commit:
- Adds content to the Connect screen based on the Figma mockups

Co-Authored-By: Amanda Beiner <aebeiner@gmail.com>

### CODE REVIEW

**Review by johnschoeman — APPROVED** [review]
LGTM

File: src/locales/en.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "_display_name": "English",
-  "about": {
-    "description": "The {{applicationName}} app is made available by {{healthAuthorityName}}",
+  "connect": {
+    "description": "The {{applicationName}} app is made available by {{healthAuthorityName}}.\n\nFor more information, visit the ",
```
Comment (aledustet) [inline src/locales/en.json:4]: I think that this text should not change if this is meant to be the same as the About screen text. This text will be replaced by configuration from each of the authorities so maybe we should leave the default as is. Probably should confirm with product.

Comment (johnschoeman) [reply src/locales/en.json:4]: We should rollback all of these language file changes and use the exact same text, as we dont have time to wait for the translations. We can updated it in a future commit.

Comment (devinjameson) [reply src/locales/en.json:4]: I see, thanks that's helpful. I'm going to merge for now just to get this through but will update in the next pass!

**Pull request merged**